modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,644 @@
+= Production Readiness Checklist 
+
+Before running a production workload on Redpanda, follow this readiness checklist to ensure that you're set up for success. Redpanda Data recommends using the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc[automated deployment instructions] in the Redpanda Ansible collection. If you cannot deploy with Ansible, use the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment.adoc[manual deployment instructions].
+
+== Level 1 production readiness
+
+The Level 1 readiness checklist helps you to confirm that:
+
+- All required defaults and configuration items are specified.
+- You have the optimal hardware setup.
+- Security is enabled.
+- You are set up to run in production.
+
+[[redpanda-license]]
+=== Load Redpanda license
```

Comment:
```suggestion
=== Redpanda license
```

Reply:
@Feediver1 I'm going to suggest tighter secondary headings, to make the right sidebar easier to read.